client/components/tinymce/index.jsx

```diff
@@ -178,6 +181,12 @@ module.exports = React.createClass( {
 
 		this.localize();
 
+		let toolbar1 = 'wpcom_add_media,';
+		if ( config.isEnabled( 'post-editor/contact-from' ) ) {
```

Comment:
This feature flag is a typo, should be `post-editor/contact-form`. It's at least consistently a typo :smile: So all references to this configuration value need to be updated.